### PR TITLE
create header (title and navigation bar)

### PR DESCRIPTION
**Pull request by raniarusly**
Current UI
Please feel free to adjust the color :D 
the bar item turns darker when cursor hover over it, like shown on the image
image

### CODE REVIEW

File: src/main/webapp/index.html [hunk 1]
```diff
@@ -4,8 +4,20 @@
...
     <title>Student Recommendations</title>
     <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="/styles/fonts.css">
+    <link rel="stylesheet" href="/styles/common.css">
   </head>
   <body>
-    <h1>Welcome to Rania and Lorna's capstone project</h1>
+    <div class="header">
+      <h1>StudentRecs</h1>
+      <h3>A recommendation system for and by students!</h3>
```
Comment (lornaarmstrong) [inline src/main/webapp/index.html]: The wording we discussed might be a bit better here - "a recommendation system for students, by students!".

File: src/main/webapp/styles/fonts.css [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+}
+/* latin */
+@font-face {
+  font-family: 'Libre Baskerville';
+  font-style: normal;
+  font-weight: 400;
+  font-display: swap;
+  src: local('Libre Baskerville'), local('LibreBaskerville-Regular'), url(https://fonts.gstatic.com/s/librebaskerville/v7/kmKnZrc3Hgbbcjq75U4uslyuy4kn0qNZaxM.woff2) format('woff2');
+  unicode-range: U+0000-00FF, U+0131, U+0152-0153, U+02BB-02BC, U+02C6, U+02DA, U+02DC, U+2000-206F, U+2074, U+20AC, U+2122, U+2191, U+2193, U+2212, U+2215, U+FEFF, U+FFFD;
+}
```
Comment (lornaarmstrong) [inline src/main/webapp/styles/fonts.css]: nit: newline needed here

**Review by lornaarmstrong — APPROVED** [review]
LGTM!

File: src/main/webapp/index.html (hunk 1 above)
Comment (raniarusly) [reply src/main/webapp/index.html]: thank you for the suggestions Lorna! :D I have just changed it accordingly

File: src/main/webapp/styles/fonts.css [hunk 3]
```diff
@@ -0,0 +1,54 @@
+/* latin-ext */
+@font-face {
+  font-family: 'Pridi';
+  font-style: normal;
+  font-weight: 400;
+  src: local('Pridi Regular'), local('Pridi-Regular'), url(https://fonts.gstatic.com/s/pridi/v5/2sDQZG5JnZLfkcWTqWgbbg.woff2) format('woff2');
+  unicode-range: U+0100-024F, U+0259, U+1E00-1EFF, U+2020, U+20A0-20AB, U+20AD-20CF, U+2113, U+2C60-2C7F, U+A720-A7FF;
+}
+/* latin */
```
Comment (KaziZubaer) [inline src/main/webapp/styles/fonts.css]: Nit: Please add a line gap :) (also please check other places in this file)

File: src/main/webapp/index.html [hunk 4]
```diff
@@ -4,8 +4,27 @@
...
+          <li><a href="#user">User Directory</a></li>
+        </ul>
+      </nav>
+    </div>
+    <div class="content">
+      <h1>TODO: add content</h1>
+    </div>
+    <footer class="footer">
+      <h2>made by Rania Rusly and Lorna Armstrong</h2>
+      <h3>STEP intern 2020</h3>
```
Comment (KaziZubaer) [inline src/main/webapp/index.html]: Umm.. these are not "headers", right? It's true that having `h2` or `h3` here gives us the desired font size and look. But if it's not actually a "heading", I will ask to use normal `div`/`p`  elements here and do the styling using CSS properties.

**Review by KaziZubaer — APPROVED** [review]
Thanks! LG(reat)TM!

**Pull request merged**